cli/buffcli/src/bin/buff/settings.rs

```diff
@@ -0,0 +1,39 @@
+use std::env;
+use config::{ConfigError, Config, File, Environment};
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize)]
+pub struct BuffGrpcServer {
+  pub url: String,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct Settings {
+  pub buff_grpc_server: BuffGrpcServer,
+}
+
+impl Settings {
+  pub fn new() -> Result<Self, ConfigError> {
+    let mut s = Config::new();
+
+    // Start off by merging in the "default" configuration file
+    s.merge(File::with_name("config/default"))?;
```

Comment:
i love the level of comments here.. can we add more comments to not copy/pasted code? ;-p